guides/common/modules/proc_installing-standalone-hammer.adoc

```diff
@@ -23,11 +45,62 @@ endif::[]
 
 +
 For more information, see {ContentManagementDocURL}Synchronizing_Repositories_content-management[Synchronizing Repositories] in _{ContentManagementDocTitle}_.
-endif::katello,orcharhino,satellite[]
+endif::[]
 ifndef::katello,orcharhino,satellite[]
 * Ensure that the CA certificate of {ProjectServer} is deployed to the truststore on the host.
 endif::[]
 
+
+.Procedure
+Select the operating system and version you are installing standalone Hammer on:
+
+* xref:#installing-standalone-hammer-el-9[{EL} 9]
+* xref:#installing-standalone-hammer-el-8[{EL} 8]
+
+[id="installing-standalone-hammer-el-9"]
+== {EL}{nbsp}9
+.Procedure
+. Enable the required repositories on the host.
+ifdef::katello,orcharhino,satellite[]
+For more information, see {ManagingHostsDocURL}Enabling_Custom_Repositories_on_Content_Hosts_managing-hosts[Enabling repositories on hosts] in _{ManagingHostsDocTitle}_.
+endif::[]
+ifdef::foreman-el[]
++
+[options="nowrap" subs="verbatim,quotes,attributes"]
+----
+# {client-package-install-el9} https://yum.theforeman.org/releases/{ProjectVersion}/el9/x86_64/foreman-release.rpm
+----
+endif::[]
+ifdef::foreman-deb[]
```

Comment:
Why are there Debian blocks under the EL 9 header? I get the impression the procedures share so little that it may be easier to write out all variants instead of merging them with many ifdefs in them.

Reply:
Writing the procedures out is a good idea, that's actually how I got things this mixed up -- I must have gotten lost in the ifdefs. I'll see what I can come up with. Thanks!

Reply:
I am also guilty as a reviewer: I made sure that every line was correct; ifdefs matched endifs; but did not see the bigger picture. :see_no_evil: